metrics/metrics.go

```diff
@@ -10,19 +10,19 @@ import (
 	"encoding/json"
 	"fmt"
 	"sync"
+	"time"
 
 	log "github.com/sirupsen/logrus"
 
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/metric"
 
-	"go.opentelemetry.io/ebpf-profiler/libpf"
 	"go.opentelemetry.io/ebpf-profiler/vc"
 )
 
 var (
 	// prevTimestamp holds the timestamp of the buffered metrics
-	prevTimestamp libpf.UnixTime32
+	prevTimestamp uint32
```

Comment:
This is to avoid circular dependency of `metrics` -> `libpf` -> `support` -> `metrics`. This is because of moving the metrics mappings to the generated file in support (to avoid adding generated files).

Reply:
We could also move `UnixTime32` to `times` or `util` package but as we don't use the `MarshalJSON` method here so it's probably not worth it just for this case.